samza-rest/src/main/java/org/apache/samza/monitor/MonitorConfig.java

```diff
@@ -96,4 +100,11 @@ public String getMonitorFactoryClass() {
   public int getSchedulingIntervalInMs() {
     return getInt(CONFIG_SCHEDULING_INTERVAL, DEFAULT_SCHEDULING_INTERVAL_IN_MS);
   }
+
+  public double getSchedulingJitterPercent() {
+    if (containsKey(CONFIG_SCHEDULING_JITTER_PERCENT)) {
+      return getDouble(CONFIG_SCHEDULING_JITTER_PERCENT);
```

Comment:
There's no getDouble(key, default) like the getInt version above?

Reply:
Switched to `getDouble(key, default) ` version.